compiler/test-resources/repl/errmsgs

```diff
@@ -64,3 +64,7 @@ scala> { def f: Int = g; val x: Int = 1; def g: Int = 5; }
 1 | { def f: Int = g; val x: Int = 1; def g: Int = 5; }
   |                ^
   |           `g` is a forward reference extending over the definition of `x`
+scala> while (((  foo ))) {}
+1 |while (((  foo ))) {}
```

Comment:
You need to shift the error message by one space after the `|`. This is a weird issue. I haven't figure it out yet